### PR TITLE
Added database for plugins

### LINKED ISSUE #938: Ranobelib doesnt work
### Source information

Ranobelib(OLD)

### Steps to reproduce

Browse -> RanobeLib
or opening any LN, which was liked from RanobeLib source

### Expected behavior

Source should work as intended

### Actual behavior

Doesn't work at all

### LNReader version

2.0.0

### Android version

Android 13

### Other details

It seems that old.ranobelib.me has a cloudflare protection, so it cant be parsed, but normal ranobelib.me doesnt(as i know)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.1.19](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Rider21**
Added database, cookie manager for plugins
Added copying from webview localstorage, sessionStorage
Depends on https://github.com/LNReader/lnreader/pull/986
------
author.today added authorization support
ranobelib rewritten for the new version of the site (closes #938)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
         "json": "tsc && node ./.js/scripts/json_plugins.js",
         "generate": "ts-node ./scripts/multisrc/generate.ts",
         "clearMultisrc": "find . -type f -wholename \"*plugins*\\[*\\]*.[t,j]s\" -delete",
+        "icon": "tsc && node ./.js/scripts/icon_reloading.js",
```
Comment (nyagami) [inline package.json:12]: there's a icon generating script in repo

File: plugins/russian/authortoday.ts [hunk 2]
```diff
@@ -4,23 +4,22 @@ import { defaultCover } from '@libs/defaultCover';
...
-const token = 'Bearer guest';
-
 class AuthorToday implements Plugin.PluginBase {
   id = 'AT';
   name = 'Автор Тудей';
   icon = 'src/ru/authortoday/icon.png';
   site = 'https://author.today';
-  version = '1.0.0';
+  apiUrl = 'https://api.author.today/v1/';
+  version = '1.0.1';
```
Comment (nyagami) [inline plugins/russian/authortoday.ts:16]: missed storage flag

Comment (Rider21) [reply plugins/russian/authortoday.ts]: The `webStorageUtilized` flag is needed to access LocalStorage, SesesionStorage

Comment (nyagami) [reply plugins/russian/authortoday.ts]: Oh, then I think we can add that lfag in app side, by checking plugin requries

Comment (Rider21) [reply plugins/russian/authortoday.ts]: I've already added this.
https://github.com/LNReader/lnreader/blob/89a1b7265734d3512e1277d727629d9d2d923487/src/plugins/types/index.ts#L70
https://github.com/LNReader/lnreader/blob/89a1b7265734d3512e1277d727629d9d2d923487/src/screens/WebviewScreen/WebviewScreen.tsx#L26
https://github.com/LNReader/lnreader/blob/89a1b7265734d3512e1277d727629d9d2d923487/src/screens/WebviewScreen/WebviewScreen.tsx#L49-L60

**Review by nyagami — APPROVED** [review]

**Comment by nyagami** [comment]
Plugins which use storage cant be used for now, they requires next version of app

**Pull request merged**